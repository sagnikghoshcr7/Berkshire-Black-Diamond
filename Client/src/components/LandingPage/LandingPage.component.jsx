import React from 'react';

import Directory from '../../components/directory/directory.component';

import { LandingPageContainer } from './LandingPage.styles';

const LandingPage = () => (
  <LandingPageContainer>
    <Directory />
  </LandingPageContainer>
);

export default LandingPage;
