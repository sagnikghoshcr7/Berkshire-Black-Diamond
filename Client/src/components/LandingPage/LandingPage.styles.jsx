import styled from 'styled-components';

export const LandingPageContainer = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
`;
