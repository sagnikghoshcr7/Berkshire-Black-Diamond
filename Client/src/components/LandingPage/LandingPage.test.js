import React from "react";
import { shallow } from "enzyme";
import LandingPage from "./LandingPage.component";

it("should render Landing Page component", () => {
  expect(shallow(<LandingPage />)).toMatchSnapshot();
});
