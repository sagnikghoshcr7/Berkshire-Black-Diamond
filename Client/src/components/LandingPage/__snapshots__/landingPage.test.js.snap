// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Landing Page component 1`] = `ShallowWrapper {}`;
